src/nvim/testdir/test_arglist.vim

```diff
@@ -74,7 +74,6 @@ func Test_argadd()
   call assert_equal(1, len(argv()))
   call assert_equal('some file', get(argv(), 0, ''))
 
-  call delete('Xargadd')
```

Comment:
8.1.0430 targets the test added in 8.1.0341 (which fails a test in test_autocmd.vim). I don't know why this was deleted.

Reply:
In Vim this is with `Test_argadd_empty_curbuf`.
Not sure what the question really is here though.
The only diff in `test_arglist.vim` is `Test_argadd_empty_curbuf` now - which will be added with v8.1.0341.

Reply:
Ah.. probably I did this wrong in ce1b4f95f.